buildenv/jenkins/variables/defaults.yml

```diff
@@ -57,6 +57,9 @@ artifactory:
   numArtifacts:
     osu: 30
     unb: 5
+  useBuildDiscarderNumArtifacts:
```

Comment:
Doesn't seem to be required. It could be removed. If `numArtifacts` for geo is not set use `build_discarder.logs` in set_artifactory_config() method:
e.g.
```
def numArtifacts = get_value(VARIABLES.artifactory.numArtifacts, geo)
if (!numArtifacts ){
    numArtifacts = get_value(VARIABLES.build_discarder.logs, pipelineFunctions.convert_build_identifier(id))
}
ARTIFACTORY_CONFIG[geo]['numArtifacts'] = numArtifacts.toInteger()
```